src/test/java/edu/kis/vh/nursery/RhymersDemo.java

```diff
@@ -31,5 +28,10 @@ public static void main(String[] args) {
 				+ ((HanoiRhymer) rhymers[3]).reportRejected());
 		
 	}
-	
+
+	private static DefaultCountingOutRhymer[] testRhymers(Rhymersfactory factory) {

```

Comment:
Extracted wrong part of the code (even the name does not make sense). The method should contain almost all code of main (except for beginning).